Makefile

```diff
@@ -14,8 +14,11 @@ shell:
 up:
 	docker-compose up
 
-flake8:
-	docker-compose run server flake8 server/
+lint:
```

Comment:
These new targets should be marked as `.PHONY` at the top of the file, as they don't depend on any files changing.